### PR TITLE
Add documentation about the query filters in NetBox dynamic inventory plugin 

### PR DESCRIPTION
**Pull request by remyleone**
##### SUMMARY

Add documentation about the query filters in NetBox dynamic inventory plugin 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

- lib/ansible/plugins/inventory/netbox.py

##### ANSIBLE VERSION

```
ansible 2.7.0.dev0
  config file = None
  configured module search path = [u'/Users/sieben/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible-2.7.0.dev0-py2.7.egg/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.15 (default, Jun 17 2018, 12:46:58) [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)]
```

### CODE REVIEW

File: lib/ansible/plugins/inventory/netbox.py [hunk 1]
```diff
@@ -65,6 +65,23 @@
   - device_roles
 query_filters:
   - role: network-edge-router
+
+# Query filters are passed directly as argument to the fetching queries.
```
Comment (samccann) [inline lib/ansible/plugins/inventory/netbox.py]: passed directly as an argument..

File: lib/ansible/plugins/inventory/netbox.py [hunk 2]
```diff
@@ -65,6 +65,23 @@
...
+# You can repeat tags in the query string.
+# Also, some more documentation and examples for query_filters would be good.
```
Comment (samccann) [inline lib/ansible/plugins/inventory/netbox.py]: "Also..."  Please remove this line.

File: lib/ansible/plugins/inventory/netbox.py [hunk 3]
```diff
@@ -65,6 +65,23 @@
...
+
+query_filters:
+  - role: server
+  - tag: web
+  - tag: production
+
+# As the documentation of NetBox specify it https://netbox.readthedocs.io/en/latest/api/overview/
```
Comment (samccann) [inline lib/ansible/plugins/inventory/netbox.py]: remove this line, or limit it to "See the NetBox documentation at https://netbox.readthedocs.io/en/latest/api/overview/."

File: lib/ansible/plugins/inventory/netbox.py [hunk 4]
```diff
@@ -65,6 +65,23 @@
...
+# the query_filters work as a logical **OR**
+#
+# In case you have custom field, prefix them with cf_ and pass the field value as with the regular NetBox query string
```
Comment (samccann) [inline lib/ansible/plugins/inventory/netbox.py]: change to 
"Prefix any custom fields with cf_ and pass the field value..."

**Review by samccann — CHANGES_REQUESTED** [review]

**Comment by remyleone** [comment]
@samccann done :)

**Review by samccann — APPROVED** [review]

**Comment by remyleone** [comment]
shipit

**Comment by samccann** [comment]
Thanks @sieben for improving the Ansible documentation!

**Comment by remyleone** [comment]
Thanks for the review :-)

**Pull request merged**